### PR TITLE
Ac404 26 deleting merchant

### PR DESCRIPTION
**Pull request by dominikn1**
I added DeletingMerchantPage inside app/ui module.

### CODE REVIEW

File: app/src/main/java/com/Found404/paypro/ui/pages/DeletingMerchantPage.kt [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                TextField(
+                    value = merchantName,
+                    onValueChange = { newValue -> merchantName = newValue }
+                )
+                Button(
+                    onClick = { /*TODO*/ },
+                    modifier = Modifier
+                        .align(End)
+                        .padding(16.dp)
+
```
Comment (mkljaic20) [inline app/src/main/java/com/Found404/paypro/ui/pages/DeletingMerchantPage.kt]: You should delete the empty space here so the code looks nicer

**Review by mkljaic20 — CHANGES_REQUESTED** [review]
You should delete an empty space left around line 80 in DeletingMerchantPage. Everything else looks good

**Review by aantonic20 — DISMISSED** [review]
Everything looks good, nice work.

**Review by timjuic — CHANGES_REQUESTED** [review]
Its a solid looking page for deleting merchants. However, according to our app sketches this should be a modal window. There is no need for an extra page for this.

Here is how you could implement it as a modal window. I asked chatgpt to make it based on your existing code. https://chat.openai.com/share/d94fa9ba-11a5-4c40-bae8-067e01039e30

**Review by jivancic20 — DISMISSED** [review]
@timjuic already covered any remark I had, so just fix what he already said.

File: build.gradle.kts
Comment (timjuic) [inline build.gradle.kts:1]: Is there a reason why this was switched to 8.2.0? This might cause issues with other modules and code

**Review by timjuic — DISMISSED** [review]
Deleting merchants appears to be working well now. Modal window and buttons look okay. It would be nice if you could just replace your inputField with our composable `PayProLabeledInput` Its single line and has already predefined styling.

There is 1 more thing that needs to be done here. And that is removing the merchant from the page that was just deleted. This could be done by just refreshing the merchant list. (getting it from the database again)

**Review by mkljaic20 — DISMISSED** [review]

**Review by mkljaic20 — APPROVED** [review]

**Review by timjuic — APPROVED** [review]

**Pull request merged**